karapace/schema_reader.py

```diff
@@ -352,19 +357,19 @@ def handle_messages(self) -> None:
 
                 assert message_key is not None
                 key = json_decode(message_key)
-            except JSONDecodeError:
+            except JSONDecodeError as exc:
                 # Invalid entry shall also move the offset so Karapace makes progress towards ready state.
```

Comment:
Comment is wrong after this change, there is no need to move offset as Karapace exits.